supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -108,6 +108,18 @@
 
 *See also*:
 
+[[offline]]
+== image:images/yes.png[yes] offline (adjective)
+
+*Description*: Use for endpoints where no network connection is present and/or all network cards are unplugged. Examples include edge devices in physically remote locations. For situations where a network is present, but devices or hosts are isolated from it, such as in restricted networks or air-gapped networks, use _disconnected_.
```

Comment:
@ShaunaDiaz Can you start this so that it's clear it's for MicroShift (just like we do for "disconnected" for openshift)?  I made a suggestion but I don't know what the official name is.

```suggestion
*Description*: In MicroShift, use _offline_ for endpoints where no network connection is present or all network cards are unplugged. Examples include edge devices in physically remote locations. For situations where a network is present, but devices or hosts are isolated from it, such as in restricted networks or air-gapped networks, use _disconnected_.
```

Reply:
Hm I don't recall the decision being just for MicroShift. Disconnected and offline really are not the same. I would think the org would want to be accurate as a whole?

Reply:
Add clarification to both definitions.